Makefile

```diff
@@ -14,30 +15,43 @@ build:
 install:
 	sudo $(INSTALL) target/release/containerd-shim-*-v1 $(PREFIX)/bin
 	
-# TODO: build this manually instead of requiring buildx
 test/out_rs/img.tar: images/image-rs/Dockerfile images/image-rs/src/lib.rs images/image-rs/Cargo.toml images/image-rs/Cargo.lock
 	mkdir -p $(@D)
-	sudo docker buildx build --platform=wasi/wasm -o type=docker,dest=$@ -t $(TEST_IMG_NAME) ./images/image-rs
+	docker build -t $(TEST_IMG_NAME) ./images/image-rs
+	docker save -o $@ $(TEST_IMG_NAME)
 
 test/out_cpp/img.tar: images/image-cpp/Dockerfile
 	mkdir -p $(@D)
-	sudo docker buildx build --platform=wasi/wasm -o type=docker,dest=$@ -t $(TEST_IMG_NAME_CPP) ./images/image-cpp
+	docker build -t $(TEST_IMG_NAME_CPP) ./images/image-cpp
+	docker save -o $@ $(TEST_IMG_NAME_CPP)
 
 test/out_dotnet/img.tar: images/aspnet/Dockerfile
 	mkdir -p $(@D)
-	sudo docker buildx build --platform=wasi/wasm -o type=docker,dest=$@ -t $(TEST_IMG_NAME_DOTNET) ./images/aspnet
+	docker build -t $(TEST_IMG_NAME_DOTNET) ./images/aspnet
+	docker save -o $@ $(TEST_IMG_NAME_DOTNET)	
 
-load: test/out_rs/img.tar test/out_cpp/img.tar test/out_dotnet/img.tar
-	sudo ctr -n $(CONTAINERD_NAMESPACE) image import $^
+test/out_spin-k/img.tar: images/spin-kitchensink/Dockerfile
+	mkdir -p $(@D)
+	docker build -t $(TEST_IMG_NAME_SPINK) ./images/spin-kitchensink
+	docker save -o $@ $(TEST_IMG_NAME_SPINK)
+
+load: test/out_rs/img.tar test/out_cpp/img.tar test/out_dotnet/img.tar test/out_spin-k/img.tar
+	sudo ctr -n $(CONTAINERD_NAMESPACE) image import test/out_rs/img.tar
+	sudo ctr -n $(CONTAINERD_NAMESPACE) image import test/out_cpp/img.tar
+	sudo ctr -n $(CONTAINERD_NAMESPACE) image import test/out_dotnet/img.tar
+	sudo ctr -n $(CONTAINERD_NAMESPACE) image import test/out_spin-k/img.tar
 
 run:
-	sudo ctr run --cni --rm --runtime=io.containerd.cehostshim.v1 docker.io/library/$(TEST_IMG_NAME) testwasm
+	sudo ctr run -d --net-host --rm --runtime=io.containerd.cehostshim.v1 docker.io/library/$(TEST_IMG_NAME) testwasm
```

Comment:
what's these two parameters?

Reply:
- `-d` runs it detached, meaning it doesn't hang the terminal, and
- `--net-host` uses the host's networking (e.g., we could ping `http://localhost:80/hello` for the spin shim demo and it would work). Considering this is just for testing purposes, I figured we could avoid using `cni` as it was sort of glitchy on my end.